src/Abstractions/NexusMods.Abstractions.Loadouts/LoadoutItem.cs

```diff
@@ -20,13 +20,13 @@ public partial class LoadoutItem : IModelDefinition
     public static readonly StringAttribute Name = new(Namespace, nameof(Name));
 
     /// <summary>
-    /// Whether the item is disabled. A disabled loadout item will not participate
-    /// in actions.
+    /// Marker to signal that the item is disabled. A disabled loadout item will
+    /// not participate in actions.
     /// </summary>
     /// <remarks>
     /// The exact meaning of a "disabled" loadout item is up to the implementations.
     /// </remarks>
-    public static readonly BooleanAttribute IsDisabled = new(Namespace, nameof(IsDisabled));
+    public static readonly MarkerAttribute IsDisabledMarker = new(Namespace, nameof(IsDisabledMarker));
```

Comment:
In this case an optional Boolean attribute could actually make sense, if we wanted to consider things like the GameFiles.
You would have True or False set states for regular mods, but could also have null (attribute not set) state for special mods like GameFiles.

I don't know if that is actually a good idea, it feels hakish and easy to misinterpret